tests/openassetio/test_logging.py

```diff
@@ -14,7 +14,7 @@
 #   limitations under the License.
 #
 """
-Tests that cover the openassetio.logging module.
+Tests that cover the openassetio.log module.
```

Comment:
[fix] This file needs renaming to `test_log.py` to match the module-under-test.

Reply:
https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/241/commits/329754435c71b5a609218b7cc5b96580c62b5c14 Fixed